pkg/client/auth/auth.go

```diff
@@ -25,17 +25,46 @@ func NewClient(conn *grpc.ClientConn) *Client {
 	}
 }
 
+// RegisterPGPPublicKeyOptions are the options to be used by RegisterPGPPublicKey function.
+type RegisterPGPPublicKeyOptions struct {
```

Comment:
nit: you could make work options work on the `authpb.RegisterPublicKeyRequest`, no need for extra structs and copying

Reply:
good point, done.